openid-federation-1_0.xml

```diff
@@ -5911,7 +5828,7 @@ Content-Type: application/json
               <t>
               Due to the large size of a Trust Chain, it may be necessary
               to use the HTTP POST method,
-              <spanx style="verb">request_uri</spanx>,
+              a <spanx style="verb">request_uri</spanx>,
```

Comment:
The `request_uri` asks the OP to fetch an external resource and this makes it easier to launch DoS attacks against Federation OPs. I propose to say `redirect_uri` MUST NOT or SHOULD NOT be used by RPs and accepted by OPs.

OIDC recognised this issue and because of that it has the OP `request_uri_parameter_supported` + `require_request_uri_registration` metadata and the RP `request_uris` metadata. So that the request_uri's can be screened before the RP can use them.

In automatic registration if a `request_uri` is used the OP must trust it blindly. If the downloaded JWT contains no Trust Chain what should the OP do then? Stop? Or resolve the `client_id`, fetching more resources?

It's good security policy to limit and minimise the number of resources at unverified URLs that need to be fetched. POSTing a `request` or using PAR should be sufficient.